docs/source/conf.py

```diff
@@ -247,3 +248,47 @@ def setup(app):
     "hankel": ("https://hankel.readthedocs.io/en/latest/", None),
     "emcee": ("https://emcee.readthedocs.io/en/latest/", None),
 }
+
+
+
+# -- Sphinx Gallery Options
+from sphinx_gallery.sorting import FileNameSortKey
+
+sphinx_gallery_conf = {
+    # path to your examples scripts
+    "examples_dirs": [
+        "../../examples/00_intro/",
+        "../../examples/01_random_field/",
+        "../../examples/02_cov_model/",
+        "../../examples/03_variogram/",
+        "../../examples/04_vector_field/",
+        "../../examples/05_kriging/",
+        "../../examples/06_conditioned_fields/",
+        "../../examples/07_transformations/",
+    ],
+    # path where to save gallery generated examples
+    "gallery_dirs": [
+        "examples/00_intro/",
+        "examples/01_random_field/",
+        "examples/02_cov_model/",
+        "examples/03_variogram/",
+        "examples/04_vector_field/",
+        "examples/05_kriging/",
+        "examples/06_conditioned_fields/",
+        "examples/07_transformations/",
+    ],
+    # Patter to search for example files
```

Comment:
Typo: "Pattern"

Reply:
Why these comments?

Reply:
fixed typo and comments for https://github.com/GeoStat-Framework/GSTools/pull/57#discussion_r370271409